benchexec/tools/map2check.py

```diff
@@ -25,35 +25,27 @@ class Tool(benchexec.tools.template.BaseTool):
 
     REQUIRED_PATHS_7_1 = ["map2check", "map2check-wrapper.py", "bin", "include", "lib"]
 
-    def executable(self):
-        # Relative path to map2check wrapper
-        if self._get_version() == 6:
-            return util.find_executable("map2check-wrapper.sh")
-        elif self._get_version() > 6:
-            return util.find_executable("map2check-wrapper.py")
-        assert False, "Unexpected version " + self._get_version()
+    def executable(self, tool_locator):
+        try:
+            executable = tool_locator.find_executable("map2check-wrapper.sh")
+            self.__version = 6
```

Comment:
I would prefer `_version`, AFAIK in Python it is uncommon to use `__field`.

Reply:
fixed in f2608c929d1e7ee7f86f1fb5f2674486fe00a4cb